.github/workflows/promote.yaml

```diff
@@ -35,6 +39,6 @@ jobs:
           base-channel: 22.04
           credentials: ${{ secrets.CHARMCRAFT_AUTH }}
           github-token: ${{ secrets.GITHUB_TOKEN }}
-          destination-channel: latest/${{ env.promote-to }}
-          origin-channel: latest/${{ env.promote-from }}
-          charmcraft-channel: latest/stable
+          destination-channel: ${{ inputs.charm-file-name }}/${{ env.promote-to }}
```

Comment:
we should use `track-name`, not `charm-file-name` as the input. The same applies at multiple places in this PR.

Reply:
whoops - don't know how I missed that, thanks